.vscode/settings.json

```diff
@@ -48,4 +48,5 @@
         },
     ],
     "references.preferredLocation": "peek",
+    "typescript.tsdk": "node_modules/typescript/lib",
```

Comment:
why add ?

Reply:
That was added automatically after linting

Reply:
okay, please delete

Reply:
I have removed it